.github/workflows/test-on-run-your-own-test.yml

```diff
@@ -0,0 +1,110 @@
+---
+# Run 'compile_and_test_for_board.py' on connected boards.
+#
+# This workflow will run on a RYOT (Run Your Own Test) machine and launch
+# all tests on all boards currently connected to that machine. This
+# workflow relies on self-hosted runners, this means building and
+# flashing happens on the self-hosted runner (all in docker). An alternative
+# approach was used in https://github.com/RIOT-OS/RIOT/pull/14600, where
+# the RYOT machine was only used for remote flashing, this is a better
+# alternative if ssh access is possible.
+#
+# Documentation:
+#
+# * Setup a RYOT machine:
+#   https://github.com/fjmolinas/riot-ryot/blob/master/setup.md
+#
+# * Requirements (already filled by a RYOT machine):
+#   * Add one or more self-hosted runners:
+#     https://docs.github.com/en/actions/hosting-your-own-runners/adding-self-hosted-runners
+#   * All required flashing tools installed
+#   * udev rules that map all BOARD to /dev/riot/tty-$(BOARD), see
+#     http://riot-os.org/api/advanced-build-system-tricks.html#multiple-boards-udev
+#   * RIOT_MAKEFILES_GLOBAL_PRE that sets PORT and DEBUG_ADAPTER_ID for each
+#     BOARD http://riot-os.org/api/advanced-build-system-tricks.html#multiple-boards-udev
+#   * Static list of connected boards or make target listing them in json format
+#     so that fromJSON can be used, see:
+#     https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#fromjson
```

Comment:
This link explains how to use `fromJSON` (which one can see below). I was asking for instructions how to provide the static list or the make target.

Reply:
is it better?